### PR TITLE
Store users's Windows credentials in dedicated table

### PR DESCRIPTION
**Pull request by willrnch**
Windows credentials are now stored in a dedicated table instead of the user one.

### CODE REVIEW

File: nanocloud/migration/users/users.go [hunk 1]
```diff
@@ -46,67 +44,131 @@ func Migrate() error {
...
-		adminlastname,
-		adminpwd,
-		true,
-	)
+	if rows.Next() {
+		return nil
+	}
 
+	rows, err = db.Query(
+		`CREATE TABLE users_windows_user (
```
Comment (dynamiccast) [inline nanocloud/migration/users/users.go:76]: Can you indent this part better please ?

File: nanocloud/models/users/users.go [hunk 2]
```diff
@@ -168,14 +165,17 @@ func CreateUser(
...
 		`INSERT INTO users
-		(id, email, activated,
-		first_name, last_name,
-		password, is_admin)
-		VALUES(
-			$1::varchar, $2::varchar, $3::bool,
-			$4::varchar, $5::varchar,
-			$6::varchar, $7::bool)
-		`, id, email, activated,
+    (id, email, activated,
```
Comment (dynamiccast) [inline nanocloud/models/users/users.go:168]: Indentation missing here too

File: nanocloud/models/users/users.go [hunk 3]
```diff
@@ -191,54 +191,63 @@ func CreateUser(
...
+	res, err := db.Query(
+		`INSERT INTO windows_users
+		(sam, password, domain)
+		VALUES ($1::varchar, $2::varchar, $3::varchar)
+		RETURNING id`,
+		sam, password, domain,
+	)
 	if err != nil {
-		return err
+		log.Error(err)
```
Comment (dynamiccast) [inline nanocloud/models/users/users.go:219]: Remaining debug log ?

**Comment by dynamiccast** [comment]
Please update go tests to match new code.

**Comment by dynamiccast** [comment]
Please update API tests to match new API defintion for user's endpoints.

**Comment by dynamiccast** [comment]
Can you tell me if this will fix this : https://github.com/Nanocloud/community/issues/26 ?

File: nanocloud/migration/users/users.go (hunk 1 above)
Comment (willrnch) [inline nanocloud/migration/users/users.go:76]: Done

File: nanocloud/models/users/users.go (hunk 3 above)
Comment (willrnch) [inline nanocloud/models/users/users.go:219]: Can be usefull.

**Comment by Gentux** [comment]
Endpoint _/api/applications/connections_ stop working with this patch. It return an empty array.

So we are unable to open a VDI

**Pull request merged**